src/pages/Search/SearchPage.tsx

```diff
@@ -151,13 +165,50 @@ function SearchPage({route}: SearchPageProps) {
                     },
                     shouldCloseModalOnSelect: true,
                 },
-            ],
+                {
+                    text: translate('export.expenseLevelExport'),
+                    icon: Expensicons.Table,
+                    onSelected: () =>
+                        // The report level export template is not policy specific, so we don't need to pass a policyID
+                        beginExportWithTemplate(CONST.REPORT.EXPORT_OPTIONS.EXPENSE_LEVEL_EXPORT, CONST.EXPORT_TEMPLATE_TYPES.INTEGRATIONS, undefined),
+                },
+            ];
+
+            // Determine if only full reports are selected by comparing the reportIDs of the selected transactions and the reportIDs of the selected reports
+            const selectedTransactionReportIDs = [...new Set(Object.values(selectedTransactions).map((transaction) => transaction.reportID))];
+            const selectedReportIDs = Object.values(selectedReports).map((report) => report.reportID);
+            const areFullReportsSelected = selectedTransactionReportIDs.length === selectedReportIDs.length && selectedTransactionReportIDs.every((id) => selectedReportIDs.includes(id));
```

Comment:
What does full report mean? Does that mean that all transactions on the report are also selected?

Reply:
Yeah, it's to differentiate whether or not we should show the "expense level export" or the "expense level export" and "report level export". The latter, we only want to show if all of the expenses on a report are selected (effectively selecting the "full" report) 

I can update this comment to make it clearer if you'd like!

Reply:
All good. Just wanted to confirm my understanding of the comment.